src/debug/HyprCtl.cpp

```diff
@@ -28,13 +28,22 @@ static std::string getWorkspaceNameFromSpecialID(const int workspaceID) {
     return workspace->m_szName;
 }
 
-std::string monitorsRequest(HyprCtl::eHyprCtlOutputFormat format) {
+std::string monitorsRequest(std::string request, HyprCtl::eHyprCtlOutputFormat format) {
+    CVarList vars(request, 0, ' ');
+    auto allMonitors = false;
+    
+    if (vars.size() > 2)
+        return "to many args";
```

Comment:
you still havent fixed this typo. "too many args" not "to many args"

Reply:
Oh, sorry I overlooked that one :see_no_evil: , I fixed it now.